interface/weno/indexrx.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2020 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+require_once("logsync.php");
+
+use OpenEMR\Rx\Weno\Container;
+
+$container = new Container();
+
+$wenoProperties = $container->getTransmitproperties();
+$provider_info = $wenoProperties->getProviderEmail();
+$urlParam = $wenoProperties->cipherpayload();          //lets encrypt the data
+$newRxUrl = "https://online.wenoexchange.com/en/NewRx/ComposeRx?useremail=";
+if ($urlParam == 'error') {   //check to make sure there were no errors
+    echo xlt("Cipher failure check encryption key");
+    exit;
+}
+?>
+<!doctype html>
+<html lang="en">
+<head>
+ <meta charset="utf-8">
+ <title><?php echo xlt('Weno eRx') ?></title>
+</head>
+<body >
+<?php
+    $urlOut = $newRxUrl . $provider_info['email'] . "&data=" . urlencode($urlParam);
```

Comment:
should also urlencode() the $provider_info['email'] variable

Reply:
No can do, this is the format required by Weno. If I change it, the URL call will fail.

Reply:
place a comment above this line stating that
(or else it will likely get changed during code walk throughs in the future)